src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -220,6 +224,60 @@ public async Task<LocationSelectViewModel> GetLocationSelectViewModel(LocationSe
             };
         }
 
+        public async Task<byte[]> GetPledgeApplicationsDownloadModel(ApplicationsRequest request)
+        {
+            var result = await _pledgeService.GetApplicationsForCsvDownload(request.AccountId, request.PledgeId);
+
+            var pledgeAppModel = new PledgeApplicationsDownloadModel
+            {
+                Applications = result.Applications?.Select(app =>
+                {
+                    var appStandard = app.Standard;
+                    var affordability = GetAffordabilityViewModel(app.Amount, app.PledgeRemainingAmount,
```

Comment:
Not clear what this is for - I thought it produced a VM for the affordability partial, so seems odd to need it here